test/integration/bitcoin_accounting_test.exs

```diff
@@ -1,4 +1,4 @@
-defmodule BitcoinAccountingTest do
+defmodule BitcoinAccounting.BitcoinAccountingIntegrationTest do
```

Comment:
Should match the filename... BitcoinAccounting.Integration.BitcoinAccountingTest...

Furthermore, I think it should be more specific about what it's testing, instead of being named after the lib's name..

Reply:
> Should match the filename... BitcoinAccounting.Integration.BitcoinAccountingTest...

will do

> Furthermore, I think it should be more specific about what it's testing, instead of being named after the lib's name..

elaborate please